pkg/volume/flocker/plugin.go

```diff
@@ -113,6 +113,10 @@ type flockerBuilder struct {
 	readOnly bool
 }
 
+func (_ *flockerBuilder) SupportsOwnershipManagement() bool {
+	return false
```

Comment:
The main question is whether we can know whether flocker volumes are ever
used in a shared mode (like NFS) or only in exclusive mode

On Wed, Oct 21, 2015 at 1:19 PM, Alexandre González <
notifications@github.com> wrote:

> In pkg/volume/flocker/plugin.go
> <https://github.com/kubernetes/kubernetes/pull/15352#discussion_r42676406>
> :
>
> > @@ -113,6 +113,10 @@ type flockerBuilder struct {
> >  	readOnly bool
> >  }
> >
> > +func (_ *flockerBuilder) SupportsOwnershipManagement() bool {
> > +	return false
>
> Hi, I spoke with @mattbates <https://github.com/mattbates> about this and
> Flocker is run (and it's mounting the volumes) as root. I don't know if
> this is going to cause any troubles in the future related with this PR. How
> could we check further? And what exactly do we need to check?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/15352/files#r42676406>.
>